shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -796,11 +796,13 @@ public void attachToFlutterEngine(@NonNull FlutterEngine flutterEngine) {
 
     // Initialize various components that know how to process Android View I/O
     // in a way that Flutter understands.
-    mouseCursorPlugin =
-        new MouseCursorPlugin(
-            this,
-            this.flutterEngine.getMouseCursorChannel(),
-            getContext());
+    if (Build.VERSION.SDK_INT >= 24) {
```

Comment:
Try to use the version constants if possible. 24 is less informative than Build.VERSION_CODES.blah